src/components/Checkboxes.test.tsx

```diff
@@ -62,6 +62,7 @@ it("renders correctly with all props", () => {
       }}
       idPrefix="example"
       onChange={(): void => console.log("on change")}
+      required={true}
```

Comment:
```suggestion
      required
```

For boolean fields, you can shorten in this way.

Reply:
I think this is required because it's a test - it has to match the expected output?